MovieNight/ActorViewController.swift

```diff
@@ -72,6 +75,15 @@ class ActorViewController: UIViewController, UITableViewDataSource, UITableViewD
     }
     
     @IBAction func didSelectActor(_ sender: Any) {
+        
+        //Show an alert in case the user have selected the limit amount of actors
+        guard actorsSelectedCount < actorsLimit  else {
```

Comment:
I think this guard is in the wrong place. It should be in the else below. Because I believe that if the user selects 3 actors and then decide to deselect one of them, this alert will be shown.